lib/ansible/modules/cloud/ovirt/ovirt_vnics_profile.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ovirt_vnics
+short_description: Module to manage vNIC profile of network in oVirt/RHV
+version_added: "2.7"
+author:
+- "Ondra Machacek (@machacekondra)"
+- "Martin Necas (@mnecas)"
+description:
+    - "Module to manage vNIC profile of network in oVirt/RHV"
+options:
+    name:
+        description:
+            - "Name of the vNIC to manage."
+        required: true
+    state:
+        description:
+            - "Should the vNIC be absent/present."
+        choices: ['absent', 'present']
+        default: present
+    data_center:
+        description:
+            - "Datacenter name where network reside."
+        required: true
+    network:
+        description:
+            - "Name of network to which is vNIC attached."
+        required: true
+    network_filter:
+        description:
+            - "Name of network filter."
+    custom_properties:
+        description:
+            - "Properties sent to VDSM to configure various hooks."
+            - "Custom properties is a list of dictionary which can have following values:"
+            - "C(name) - Name of the custom property. For example: I(hugepages), I(vhost), I(sap_agent), etc."
+            - "C(regexp) - Regular expression to set for custom property."
+            - "C(value) - Value to set for custom property."
+    qos:
+        description:
+            - "Name of Quality of Service."
+    port_mirroring:
+        description:
+            - "Boolean, sets usage of port mirroring."
+        type: bool
+    pass_through:
+        description:
+            - "String which enables being directly attached to a VF."
+        choices: ['disabled', 'enabled']
+    migratable:
```

Comment:
Please use proper docuemntation for all parameters as is written in API model:

http://ovirt.github.io/ovirt-engine-api-model/4.2/#types/vnic_profile